modules/price/src/Price.php

```diff
@@ -37,7 +37,7 @@ public function __construct($amount, $currency_code) {
   /**
    * Gets the decimal amount.
    *
-   * @return string
+   * @return float
```

Comment:
Well, it's definitely a string cause that's what bcmath consumes. We want to start using the Calculator class instead of operators anyway. (as soon as we merge the Calculator class, that is. Wanted to see our needs evolve first)